src/contrib/cluster/Akka.Cluster.Sharding/PersistentShardCoordinator.cs

```diff
@@ -1346,26 +1369,26 @@ protected override bool ReceiveRecover(object message)
                     switch (evt)
                     {
                         case ShardRegionRegistered _:
```

Comment:
Make sure all of the `Recovery` methods are updated to support the new `isRecovering` flag.